releasenotes/notes/0.10/0-10-release-8c37dadcc1c82fcc.yaml

```diff
@@ -1,66 +1,66 @@
-      noise models and inserting them into circuits.
+    * Support of user-created transpiler passes for defining custom gate errors and
+      noise models, and inserting them into circuits.
+
 features:
   - |
-    Adds support of QASM 3.0 control-flow instructions introduced in Qiskit-Terra
-    0.19.0. Supported instructions are :class:`qiskit.circuit.ForLoopOp`,
-    :class:`qiskit.circuit.WhileLoopOp`, :class:`qiskit.circuit.ContinueLoopOp`,
-    :class:`qiskit.circuit.BreakLoopOp`, :class:`qiskit.circuit.IfElseOp`.
+    Added support of QASM 3.0 control-flow instructions introduced in Qiskit-Terra
+    0.19.0. Supported instructions are :class:`~qiskit.circuit.ForLoopOp`,
+    :class:`~qiskit.circuit.WhileLoopOp`, :class:`~qiskit.circuit.ContinueLoopOp`,
+    :class:`~qiskit.circuit.BreakLoopOp`, :class:`~qiskit.circuit.IfElseOp`.
   - |
-    Adds a batched shot simulation optimization for GPU simulations. This
-    optional will use available memory on 1 or more GPUs to run multiple
+    Added a batched-shot simulation optimization for GPU simulations. This
+    optional feature will use available memory on 1 or more GPUs to run multiple
     simulation shots in parallel for greatly improved performance on
     multi-shot simulations with noise models and/or intermediate measurements.
 
-    This option is enabled by default when using ``device="GPU"`` for the
-    :class:`~qiskit.providers.aer.AerSimulator` for ``"statevector"`` and
-    ``"density_matrix"`` simulation methods and can be disabled by setting the
-    ``batched_shots_gpu=False`` in the simulator options.
+    This option is enabled by default when using ``device="GPU"`` and a
+    simulation ``method`` of either ``"statevector"`` or ``"density_matrix"``
+    with the :class:`~qiskit.providers.aer.AerSimulator`.  It can be disabled by
+    setting ``batched_shots_gpu=False`` in the simulator options.
 
-    This optimization is most benefical for small to medium numbers of qubits
+    This optimization is most beneficial for small to medium numbers of qubits
     where there is sufficient GPU memory to run multiple simulations in
-    parallel. The maximum number of activate circuit qubits for enabling this
-    optimization can be configued using the ``batch_shots_gpu_max_qubits``
+    parallel. The maximum number of active circuit qubits for enabling this
+    optimization can be configured using the ``batch_shots_gpu_max_qubits``
     simulator option. The default value of this option is 16.
   - |
-    Adds the new option ``max_shot_size`` option to a cutom executor for
-    run multiple shots execution of a noisy circuit in parallel. 
-
-    For example configuring ``max_shot_size`` with a custom executor 
+    Added the new ``max_shot_size`` option to a custom executor for
+    running multiple shots of a noisy circuit in parallel.
 
-    .. code-block:: python
+    For example configuring ``max_shot_size`` with a custom executor::
```

Comment:
This is short hand for code block? What about python syntax highlighting?

Reply:
exactly - `::` is short-hand for `code-block` using the default language (and it puts a `:` at the end of the paragraph).  Our default language is Python, so they get Python highlighting.